starboard/elf_loader/exported_symbols.cc

```diff
@@ -157,12 +157,16 @@ ExportedSymbols::ExportedSymbols() {
 #if SB_API_VERSION < 16
   REGISTER_SYMBOL(SbDirectoryCanOpen);
 #endif  // SB_API_VERSION < 16
+#if SB_API_VERSION < 17
   REGISTER_SYMBOL(SbDirectoryClose);
+#endif  // s SB_API_VERSION < 17
 #if SB_API_VERSION < 16
   REGISTER_SYMBOL(SbDirectoryCreate);
```

Comment:
Why are we gating some directory apis with SB16 and some with SB17? what's the criteria?

Reply:
SB16 froze a couple of weeks ago. We can no longer make starboard changes since partners are locked into the currently released starboard version.